### PR TITLE
added particle coagulation in parcel class

### LINKED ISSUE #47: add particle to particle interacations in parcel class
https://github.com/uncscode/particula/blob/85ba3f80b2f6d5b7d87c0e9f1a848d1bb9d86e94/aerosol_dynamics/parcel.py#L98

### PR DESCRIPTION
**Pull request by Gorkowski**
Fixes #47

### CODE REVIEW

File: particula/aerosol_dynamics/parcel.py [hunk 1]
```diff
@@ -1,21 +1,20 @@
-"""class for creating a air parcel to put particles and gases in.
+"""class for creating an air parcel to put particles and gases into.
 """
 
 import numpy as np
-from particula import u
```
Comment (ngam) [inline particula/aerosol_dynamics/parcel.py:5]: Can you bring this back and do some unit checks as decorators?

**Review by ngam — CHANGES_REQUESTED** [review]

File: particula/aerosol_dynamics/parcel.py (hunk 1 above)
Comment (Gorkowski) [reply particula/aerosol_dynamics/parcel.py:5]: We can talk about how we want to handle units, based on @wkchuang pull draft. I ran into unit errors/mismatches when assigning units in the inputs (so I removed them). I think it came from reloading the module and having inconsistent instances of unitreg, but not completely sure.

Comment (Gorkowski) [reply particula/aerosol_dynamics/parcel.py:5]: I added back the unit check for radius. The mismatch was due to multiple instances of unitreg during modifying parcel.py. 
We do not need 'from particula import u' in as either the units will be in the input or the particle class will assign them, no units get created in parcel class. We don't need 'u' to do the unit checks, with the current try-exempt check being used.

**Review by ngam — APPROVED** [review]

File: particula/aerosol_dynamics/parcel.py (hunk 1 above)
Comment (ngam) [reply particula/aerosol_dynamics/parcel.py:5]: Maybe we should consider taking all units out and place them in the test suite. I think that makes significantly more sense to me than having units around in the actual routines. We will keep talking about this in PR #84

**Pull request merged**